vehicle-model-lifecycle/src/download_vspec.py

```diff
@@ -40,6 +39,14 @@ def is_uri(path: str) -> bool:
     """
     return re.match(r"(\w+)\:\/\/(\w+)", path) is not None
 
+def download_unit_file(uri: str, local_file_path: str) -> None:
```

Comment:
yes because if you download from a github path then you get a blob path. Since vehicle-signal-specification only released units.yaml since 4.1 (which is not supported yet by us) and not in release 4.0 I needed to add this to get the file out of some json structure.

Reply:
Would be even more of a reason to put a 4.0 unit file directly in this repo rather than extracting it from a github blob :-)